sympy/combinatorics/fp_groups.py

```diff
@@ -76,6 +76,27 @@ def __new__(cls, fr_grp, relators):
     def free_group(self):
         return self._free_group
 
+    @property
+    def relators(self):
+        return tuple(self._relators)
```

Comment:
I don't think so. And I don't think there is any reason to have `self._free_group` as well as `self.free_group`. Should I change both of them?